README.md

```diff
@@ -29,7 +29,16 @@ In `snack/main.go`, type in random characters and save. Tilt will start a build,
 Fix the error and move on.
 
 #### Startup Error
-Kubernetes Pods can get into `CrashLoopBackOff` when they can't startup, e.g. because they can't find a necessary resource file.
+Kubernetes Pods can get into `CrashLoopBackOff` when they can't startup, e.g. because they can't find a necessary resource file. Open `snack/main.go` and find the first `log.Fatal`; uncomment it and save.
```

Comment:
nice! but maybe a bit of a red herring in re: the error we're telling them to trip. wdyt of this?
```suggestion
Kubernetes Pods can get into `CrashLoopBackOff` when they crash multiple times in a row, e.g. if they always crash on startup because of a runtime error. Open `snack/main.go` and find the first `log.Fatal`; uncomment it and save.
```